### PR TITLE
Auth 0

### PR DESCRIPTION
**Pull request by tehnools**

### CODE REVIEW

File: Controllers/CasesController.cs [hunk 1]
```diff
@@ -5,104 +5,108 @@
...
+      _context = context;
+    }
 
-        public CasesController(BackendContext context)
-        {
-            _context = context;
-        }
+    // GET: api/Folders
+    [HttpGet]
+    public async Task<ActionResult<IEnumerable<Folder>>> GetFolders()
```
Comment (williamparry) [inline Controllers/CasesController.cs]: Why are you replacing Cases into Folders in the CasesController?

File: .gitignore [hunk 2]
```diff
@@ -8,6 +8,9 @@
 *.user
 *.userosscache
 *.sln.docstates
+.env
+Migrations
```
Comment (williamparry) [inline .gitignore]: Shouldn't we commit the Migrations?

File: Startup.cs [hunk 3]
```diff
@@ -11,50 +11,58 @@
...
-        {
-            Configuration = configuration;
-        }
-
-        public IConfiguration Configuration { get; }
-
-        // This method gets called by the runtime. Use this method to add services to the container.
-        public void ConfigureServices(IServiceCollection services)
-        {
+      Configuration = configuration;
```
Comment (williamparry) [inline Startup.cs]: We should align our formatting.

File: .gitignore (hunk 2 above)
Comment (tehnools) [reply .gitignore]: Removed Migrations from ignore

File: Startup.cs (hunk 3 above)
Comment (tehnools) [reply Startup.cs]: realigned the spaces to keep as close to the original spacing.

File: Controllers/CasesController.cs (hunk 1 above)
Comment (tehnools) [reply Controllers/CasesController.cs]: I am not sure what happened here but even some of my auth0 changes in the docker container didn't transfer over to the local folder.

File: Controllers/CasesController.cs [hunk 4]
```diff
@@ -5,104 +5,108 @@
...
-        private readonly BackendContext _context;
+      _context = context;
+    }
 
-        public CasesController(BackendContext context)
-        {
-            _context = context;
-        }
+    // GET: api/Cases
+    [HttpGet]
```
Comment (williamparry) [inline Controllers/CasesController.cs]: All of these methods should be authorised

Comment (tehnools) [reply Controllers/CasesController.cs]: Ok i'll get these authorized.

**Comment by tehnools** [comment]
#close

**Pull request closed without merging**